mobile/lib/modules/home/ui/asset_grid/thumbnail_image.dart

```diff
@@ -136,10 +136,8 @@ class ThumbnailImage extends StatelessWidget {
           tag: isFromDto
               ? '${asset.remoteId}-$heroOffset'
               : asset.id + heroOffset,
-          child: ImmichImage(
+          child: ImmichImage.thumbnail(
             asset,
```

Comment:
`useGrayBoxPlaceholder` has to be passed to the factory constructor and should be passed down here. Without this, the main timeline do not show any grey placeholder while loading the thumbnails.